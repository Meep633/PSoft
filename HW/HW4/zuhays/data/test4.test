l
data/rpi_map_simple_nodes.csv
data/rpi_map_simple_edges.csv
r
0
3
r
0
7
r
0
8
r
1
1
r
what

r
invalid
invalid
q