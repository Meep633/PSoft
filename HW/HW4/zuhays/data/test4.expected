Building file name, followed by Enter: Building paths file name, followed by Enter: Successfully loaded.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from 0 to 3:
	Walk East to (1)
	Walk East to (Intersection 2)
	Walk East to (3)
Total distance: 3.000 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from 0 to 7:
	Walk South to (4)
	Walk South to (Intersection 5)
	Walk East to (6)
	Walk South to (7)
Total distance: 4.000 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: There is no path from 0 to 8.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from 1 to 1:
Total distance: 0.000 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [what]
Unknown building: []
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [invalid]
